.gitignore

```diff
@@ -13,27 +13,31 @@ build/
 !default.mode2v3
 *.perspectivev3
 !default.perspectivev3
-xcuserdata
+**/xcuserdata
 *.xccheckout
 *.moved-aside
-DerivedData
+*DerivedData
 *.hmap
 *.ipa
 *.xcuserstate
 
 # Android/IntelliJ
 #
-build/
-.idea
-.gradle
-local.properties
+**/.idea
```

Comment:
looks like you are missing `build/` dir for Android. Please double check.

Reply:
there is one in line 7, please confirm